packages/@aws-cdk/aws-logs/test/integ.metricfilter-dimensions.ts

```diff
@@ -0,0 +1,34 @@
+import { App, RemovalPolicy, Stack, StackProps } from '@aws-cdk/core';
+import { IntegTest } from '@aws-cdk/integ-tests';
+import { FilterPattern, LogGroup, MetricFilter } from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const logGroup = new LogGroup(this, 'LogGroup', {
+      removalPolicy: RemovalPolicy.DESTROY,
+    });
+
+    /// !show
```

Comment:
```suggestion
```

We don't use this anymore. If you want this added to the README, you'll need to add it manually.